src/libs/API.js

```diff
@@ -159,15 +156,12 @@ Network.registerResponseHandler((queuedRequest, response) => {
         // There are some API requests that should not be retried when there is an auth failure like
         // creating and deleting logins. In those cases, they should handle the original response instead
         // of the new response created by handleExpiredAuthToken.
-        const shouldRetry = lodashGet(queuedRequest, 'data.shouldRetry');
-        if (!shouldRetry || unableToReauthenticate) {
+        if (!shouldRetry(queuedRequest) || unableToReauthenticate) {
```

Comment:
We usually use `should*` for booleans and not functions.

Reply:
Renamed to `isRequestRetrieable`